Products/CMFPlone/browser/templates/ajax_main_template.pt

```diff
@@ -16,7 +16,7 @@
           portal_url python:portal_state.portal_url();
           checkPermission python:context.restrictedTraverse('portal_membership').checkPermission;
           ajax_include_head python:request.get('ajax_include_head', False);
-          ajax_load python:False;"
+          ajax_load python:True;"
```

Comment:
I have no idea why this `ajax_main_template` has `ajax_load=False` in the first place. This variable seems really obscure defined like this.